src/lib/components/SplitPane/Resizer/Resizer.tsx

```diff
@@ -0,0 +1,28 @@
+import * as S from './Resizer.styled';
+import type { Direction, HexColor } from '../../../types/';
+
+type ResizerProps = {
+  id: number;
+  direction: Direction;
+  thickness: number;
+  themeColor: HexColor;
+};
+
+const Resizer = (props: ResizerProps) => {
+  const { id, direction, thickness, themeColor } = props;
+
+  return (
+    <S.Container
+      $direction={direction}
+      $thickness={thickness}
+      $themeColor={themeColor}
+      data-resizer-id={id}
+    >
+      {Array.from({ length: 3 }).map(() => (
+        <S.Diamond $size={thickness / 3} $themeColor={themeColor} />
+      ))}

```

Comment:
요렇게 쓰는 거하고 차이가 있을까요?

```suggestion
      {Array.from({ length: 3 }, () => (<S.Diamond $size={thickness / 3} $themeColor={themeColor} />)}
```

추가적으로 꼭 필요하다고 볼수는 없지만 id 값도 전달해줬으면 좋겠네요!